packages/grid/stories/grid-vuu.stories.tsx

```diff
@@ -0,0 +1,126 @@
+import {
```

Comment:
Not sure what we should do with this story. It's not going to work for most people. Also, shouldn't be displayed under Lab.

I suggest remove this one altogether.